tests/components/shelly/test_init.py

```diff
@@ -448,7 +450,9 @@ async def test_entry_unload_not_connected_but_we_think_we_are(
         )
         assert entry.state is ConfigEntryState.LOADED
 
-        assert (state := hass.states.get("switch.test_switch_0"))
+        assert (
+            state := hass.states.get("switch.test_name_12_34_56_78_9a_be_test_switch_0")
```

Comment:
This is related to the BLE scanner device (its name is `Test name (12:34:56:78:9A:BE)`). I don't understand why HA uses this device name to generate Shelly device entity names, and not the Shelly device name.